assets/js/googlesitekit/widgets/datastore/widgets.js

```diff
@@ -333,7 +333,7 @@ export const selectors = {
 
 				return (
 					intersection( widget.modules, allowedModules ).length ===
-					allowedModules.length
+					widget.modules.length
```

Comment:
This change is required to fix the implementation of `getWidgets` - see https://github.com/google/site-kit-wp/issues/5161#issuecomment-1127946815 for more details.